vizro-core/docs/pages/explanation/schema.md

```diff
@@ -0,0 +1,156 @@
+
+This means you can configure a Vizro dashboard according to a set of constraints that are defined in the schema. The configuration language that you choose is secondary - it can be via Python, but also via JSON or YAML. Exactly this is shown in [our showcase of configuration options](../user-guides/dashboard.md#use-dashboard-configuration-options).
+
+=== "This json..."
+    ```json
+    {
+      "pages": [
+        {
+          "components": [
+            {
+              "figure": {
+                "_target_": "scatter",
+                "color": "species",
+                "data_frame": "iris",
+                "x": "sepal_length",
+                "y": "petal_width"
+              },
+              "type": "graph"
+            },
+            {
+              "figure": {
+                "_target_": "histogram",
+                "color": "species",
+                "data_frame": "iris",
+                "x": "sepal_width"
+              },
+              "type": "graph"
+            }
+          ],
+          "controls": [
+            {
+              "column": "species",
+              "type": "filter"
+            }
+          ],
+          "title": "My first dashboard"
+        }
+      ]
+    }
+    ```
+
+=== "maps to this Dashboard"
+    [![Dashboard]][dashboard]
+
+!!! note
+    The Vizro schema is not yet complete. This means that it does not yet define everything that we consider a core and supported functionality of Vizro. The most prominent example of this is the [`CapturedCallable`][vizro.models.types.CapturedCallable] - the objects you insert into many models such as `vm.Graph`, `vm.Table` or `vm.Action`, often into the `figure` argument.
+
+    This does not mean that it cannot be configured in JSON - in fact, in the config above we have used the following JSON:
+
+    ```json
+    {
+      "figure": {
+        "_target_": "histogram",
+        "color": "species",
+        "data_frame": "iris",
+        "x": "sepal_width"
+      },
+      "type": "graph"
+    }
+    ```
+
+    It does mean however that e.g. `figure` does not appear in the schema - and as such is not clearly defined as part of the Vizro framework, although it should. We are working on completing the Vizro schema in the future.
+
+## The role of Pydantic
+
+The Vizro framework is powered by [Pydantic](https://docs.pydantic.dev/latest/) - the most widely used data validation library for Python. Pydantic is acts as the glue that connects Vizro's JSON schema to it's emergent manifestation - a React front end served by a Flask backend, facilitated by Dash.
+
+One core advantage of Pydantic is that it emits a [JSON schema](https://blog.postman.com/what-is-json-schema) - so that our models such as `vm.Page` or `vm.Filter` can easily be translated into a well-defined JSON schema. Beyond this capability, it also has support for custom validation beyond the functionality of the JSON schema.
```

Comment:
```suggestion
One core advantage of Pydantic is that it emits a [JSON schema](https://blog.postman.com/what-is-json-schema) so that our models, such as `vm.Page` or `vm.Filter`, can easily be translated into a well-defined JSON schema. Beyond this capability, it also has support for custom validation beyond the functionality of the JSON schema.
```